### PR TITLE
Chaos 78 integrate create organisation with backend

### PR DESCRIPTION
**Pull request by stapletm**

### CODE REVIEW

File: frontend/src/components/AdminSideBar/index.js [hunk 1]
```diff
@@ -37,11 +39,15 @@ const AdminSidebar = ({
...
+  const onUpload = async () => {
     // FIXME: CHAOS-55, send to the backend
     if (uploadedImage.image && inputText) {
       // FIXME: CHAOS-55, backend request should return new id, this method obv flawed (also floored)
-      const newID = Math.floor(Math.random() * 1000);
+      const imgUrl = base64ToBytes(
+        (await fileToDataUrl(uploadedImage.image)).split(",")[1]
+      );
+      const newID = await createOrganisation(inputText, imgUrl);
+      // const newID = Math.floor(Math.random() * 1000);
```
Comment (zaxutic) [inline frontend/src/components/AdminSideBar/index.js:50]: wtf

**Review by zaxutic — CHANGES_REQUESTED** [review]
delete comment smh cringe

File: frontend/src/components/AdminSideBar/index.js (hunk 1 above)
Comment (stapletm) [reply frontend/src/components/AdminSideBar/index.js:50]: thank you

**Review by zaxutic — APPROVED** [review]
epic

**Pull request merged**